deployments/common/zeppelin/sql/auth.sql

```diff
@@ -0,0 +1,27 @@
+USE zeppelin;
+CREATE TABLE users (username TEXT, password TEXT, password_salt TEXT);
+CREATE TABLE user_roles (username TEXT, role_name TEXT);
+CREATE TABLE user_permissions (username TEXT, permission TEXT);
+GRANT ALL PRIVILEGES ON zeppelin.users TO 'zeppelin'@'localhost';
+GRANT ALL PRIVILEGES ON zeppelin.user_roles TO 'zeppelin'@'localhost';
+GRANT ALL PRIVILEGES ON zeppelin.user_permissions TO 'zeppelin'@'localhost';
+
+
+INSERT INTO users (username, password) VALUES ('gaiauser', '$shiro1$SHA-256$500...........R0GxWVAH028tjMyIkbKmMDW2E0=');
+INSERT INTO users (username, password) VALUES ('dcr', '$shiro1$SHA-256$500000...........bp5aQVQtw6kmVMUlENoGkrBPjCDqWOs=');
+INSERT INTO users (username, password) VALUES ('nch', '$shiro1$SHA-256$500000$MPs..............5lBwoKj7LtnBMUZJp4XmCyBv9yvMZrM=');
+INSERT INTO users (username, password) VALUES ('zrq', '$shiro1$SHA-256$50000...............Pm6tS4XfOETCwEwI8Ri5FE8GfM2uBRQ=');
+INSERT INTO users (username, password) VALUES ('stv', '$shiro1$SHA-25.....................sAkP3jjvbeq1KHgieb00pIizAM=');
+INSERT INTO users (username, password) VALUES ('admin', '$shiro1$SHA-256$50000.................p+LFeSFqvBNMKVpYMVMCc+cE8/rrTQI=');
+INSERT INTO users (username, password) VALUES ('yrvafhom', '$shiro1$SHA-256$500000$b/..............C9diAs6AUdv/29qjtsSLqmWUQ=');
```

Comment:
Redacting the (encrypted) passwords is good for security, but does this mean this script doesn't actually insert the right passwords ?

Reply:
Is it safe to include the encrypted passwords here? I was assuming no, in which case I don't know what a good solution is until we can store the passwords on a Vault or equivalent

Reply:
Good question. They are encrypted, so what could a bad actor do with them ?

Reply:
I don't really know enough about encryption algorithms to give a good answer, but I'm wondering if attackers could use the hashes to spot patterns that may allow them to narrow down what hashing algorithm is being used? In general it just feels wrong but I'd have to do some research on it to give a better answer.

Reply:
Quick look in Google found this
https://security.stackexchange.com/questions/246209/is-it-safe-to-publish-the-hash-of-my-passwords

Specifically
https://security.stackexchange.com/a/246248

> Q. How unsafe would be to publish the hash of my passwords?
> A. Very unsafe. Irrespective of the algo used, please never share the passwords or the hash of it on open web. 

So probably right to keep them safe.

Reply:
As a short term fix. How about put the accounts file in a directory on your laptop and mount it as a volume in the client container ? That would enable us to deploy this now and work on the secrets later.

Reply:
Ok this is what I've done in my most recent tests.